frontend/src/app.tsx

```diff
@@ -111,3 +124,38 @@ function AppFrame({ children }: AppFrameProps) {
     </div>
   )
 }
+
+const logger = createLogger('Errorboundary')
+
+class ErrorBoundary extends Component<
```

Comment:
question: as we discovered recently once this is hit there is no way to recover. We do have a way to manually recover from this?

Reply:
Added a button reload the app.